docs/modules/ROOT/pages/release-notes.adoc

```diff
@@ -0,0 +1,28 @@
+= Release Notes
+
+This release introduced LDAP Security Provider support for Management Center CR.
```

Comment:
Not sure if this line is required here since we mention it on the next line - new features.

Reply:
Yes, I think it's ok. We made it multiple times before. The line under Release Notes it's a kinda of summary of what was in the release and under New Feature it's details with the PR number.